e2etest/contract_staking_test.go

```diff
@@ -1342,7 +1342,7 @@ func TestContractStaking(t *testing.T) {
 		receipts, blk := writeContract(bc, sf, dao, ap, []*callParam{&param}, r)
 		r.Len(receipts, 1)
 		r.EqualValues(iotextypes.ReceiptStatus_Success, receipts[0].Status)
-		buckets, err := indexer.Buckets()
+		buckets, err := indexer.Buckets(blk.Height())
```

Comment:
if `blk.Height() == indexer height + 1`, it should fire an error here

Reply:
block has been committed after `writeContract`